README.md

```diff
@@ -1,26 +1,178 @@
+                print("authenticated successfully: \(account)")
+            case .Failure(let error):
+                print("failure authenticating: \(error)")
+            }
+        }
+        
+        return true
+    }
+```
+
+### Saved Accounts
+
+`AuthenticationController` saves the accounts it successfully authenticates in the Keychain.  The next time your application opens, you should first attempt to load a previously authenticated account before prompting the user to authenticate.
+
+```Swift
+do {
+	if let account = try authenticationController.loadSavedAccount() {
+		print("account loaded successfully: \(account)"
+	} else {
+		print("no saved account found, authenticate...")
+	}
+}
+catch let error {
+	print("error loading account: \(error)")
+}
+```
+
+## Interacting with Vimeo
+
+You're initialized, you're configured, you're authenticated.  Now you're ready to rock!  Let's start hitting some endpoints. 🤘
+
+### Making Requests
+
+We represent each interaction with the Vimeo API as an instance of `Request`.  Each one holds its own understanding of HTTP method, URL path, parameters (if any), expected model object type, caching behavior, and (if desired) retry-after-failure behavior.  That's a lot to grasp up front, so let's step back and look at the simplest request we can make:
+
+```Swift
+let videoRequest = Request<VIMVideo>(path: "/videos/45196609")
+```
+
+There are two critical takeaways here:
+- `<VIMVideo>`: the generic type of the model object.  This is what request will return in the `Response` if it's successful.
+- `path: "..."`: the API endpoint we want to call, you can see a whole host of other options at [Vimeo API Endpoints](https://developer.vimeo.com/api/endpoints)
+
+By declaring the expected model object type, we can ensure that the both the request and the parsing of the JSON is successful, and we can guarantee that our `Response` precisely matches this expectation.  All potential Vimeo models are already implemented in **VimeoNetworking**, and they're available for use in your own implementation.  Model classes are all named with the `VIM` suffix: `VIMUser`, `VIMChannel`, `VIMCategory`, and so on.
+
+### Handling Responses
+
+After we send that response, we'll get a `Result` enum back.  This could be either a `.Success` or a `.Failure` value.  `.Success` will contain a `Response` object, while `.Failure` will contain an `NSError`.  Switch between these two cases to handle whichever is encountered:
+
+```Swift
+vimeoClient.request(videoRequest) { result in 
+	switch result {
+	case .Success(let response: Response):
+		let video: VIMVideo = response.model
+		print("retrieved video: \(video)")
+	case .Failure(let error: NSError):
+		print("error retrieving video: \(error)"
+	}
+}
+```
+
+
+### Collections
+
+One neat **ProTip**: arrays of model objects are also valid generic types of `Request`.  In this case, the `model` property of your `Response` will be an array of parsed models of the expected type.
+
```

Comment:
another minor thing you can take or leave, this was super clear after reading the code. i wonder if this could be phrased slightly differently in more narrative form like, when 'you make a request like this, you can expect a response like this'  -